### ISSUE TITLE
Problem with www.digitalattackmap.com when HTTPS Everywhere is enabled

### ISSUE THREAD

**Issue by moonlies** [EFForg/https-everywhere#7939]
Some sites do not show javascript content when https-everywhere (version 5.2.8 for Firefox) is enabled. I only encountered this behaviour in Firefox (version 50.1.0). The chromium addon seems to work nicely, at least for the tested site: digitalattackmap.com.

**Comment by jeremyn** [comment]
Thanks for the report. With HTTPS Everywhere enabled on Firefox 50.1.0 on Ubuntu, at http://www.digitalattackmap.com , I get this message:

> Unfortunately, an error occurred while loading geographic data. You might need to upgrade your browser.

I get the following console errors:

```
Using //@ to indicate sourceMappingURL pragmas is deprecated. Use //# instead[Learn More]  jquery.min.js:1
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://www.gstatic.com/ddos-viz/attacks_v2.json. (Reason: CORS header ‘Access-Control-Allow-Origin’ does not match ‘(null)’).  (unknown)
TypeError: path is null[Learn More] www.digitalattackmap.com:94:11
TypeError: path is null[Learn More] www.digitalattackmap.com:94:11
TypeError: path is null[Learn More]  www.digitalattackmap.com:94:11
TypeError: path is null[Learn More] www.digitalattackmap.com:94:11
TypeError: path.setAttribute is null[Learn More] www.digitalattackmap.com:94:11
TypeError: path is null[Learn More] www.digitalattackmap.com:94:11
```

The `TypeError`s repeat very fast. I don't get this problem on Chromium 55.0.2883.87 on Ubuntu.

I've updated the issue description to be more specific.

**Comment by gitarra** [comment]
Another CORS issue..

**Comment by krmboya** [comment]
Indeed, when I disable Https Everywhere for Google APIs on the page it works. 

It appears this had been previously fixed here https://github.com/EFForg/https-everywhere/pull/6634#event-770051769 but the URL changed from `http://www.gstatic.com/ddos-viz/attacks.json` to `http://www.gstatic.com/ddos-viz/attacks_v2.json`.

Unfortunately I get some errors when I try run a standalone firefox profile or build an xpi according to the README instructions therefore cannot test out a fix (Ubuntu 14.04)

**Comment by jeremyn** [comment]
I don't know about your Firefox profile issues, but I find it convenient to work with profiles using `about:profiles` instead of `firefox -p`. Maybe that will help you.

For testing, you don't need to create an xpi to test a ruleset in the browser, instead you can copy the new ruleset to a special directory. See [here](https://www.eff.org/https-everywhere/rulesets#testing) for more information.
